README.md

```diff
@@ -7,36 +7,72 @@
 
 ## Description
 
-A flexible front end to plate bases pipelines in Sequencescape
+A flexible front end to plate bases pipelines in Sequencescape.
 
 ## Initial Setup
 
-1. In Limber, connect to Sequencescape to configure required data:
-```shell
-  bundle exec rake config:generate
-```
+Steps must be followed in either the Sequencescape repository or this Limber
+repository, as indicated:
 
-2. In Sequencescape, configure all Limber required data:
-```shell
-  bundle exec rake limber:setup
-```
+1. In Limber, ensure the appropriate version of Ruby is installed. The command
+   here is for `rbenv` but you may want to use a different Ruby version manager.
+   If the command fails, you might need to check the list of available Ruby
+   versions using `rbenv install --list-all` and use the one that most closely
+   matches the contents of `.ruby-version`:
 
-3. Start Sequencescape (will start in 3000):
-```shell
-  bundle exec rails s
-```
+   ```shell
+   rbenv install
+   ```
 
-4. Start the delayed job processor
-```shell
-  bundle exec rake jobs:work
-```
+2. In Limber, make the Bundler gem install the dependencies for this project:
 
-5. In Limber, start Limber (will start in 3001):
-```shell
-  bundle exec rails s
-```
+   ```shell
+   bundle install
+   ```
+
+3. In Limber, install the yarn dependencies:
+
+   ```shell
+   yarn install
+   ```
+
+4. In Sequencescape, configure all Limber required data:
+
+   ```shell
+   bundle exec rake limber:setup
+   ```
+
```

Comment:
The previous instructions were advising running config:generate before limber:setup as it better ensures that limber remains the authority on plate purposes. The alternative approach does work, but *should* be showing warnings. (I suspect they are buried in the rest of the logs)

Reply:
Background: Annoyingly we need to have request types validate the plate purpose of their source wells, as otherwise we get a constant stream of user errors. Currently this is via an association on request type (acceptable plate purposes) which means request type setup is dependent to the plate purposes.

I *really* want to decouple these, but haven't had the chance.

Reply:
I'll move this step down in the sequence.  I wasn't sure whether we'd prefer this to be done before SS gets fired up, but it sounds like that can't be avoided.

Reply:
Yeah, its pain enough that the 'wrong' way will still work, but I figured the instructions should encourage the 'correct' approach.